LanguageMerger/LanguageFiles/tfg/en_us/ore_veins.json

```diff
@@ -47,7 +47,31 @@
 "ore_vein.tfg.surface_copper": "Chalcopyrite & Realgar",
 "ore_vein.tfg.surface_sphalerite": "Sphalerite & Sulfur",
 "ore_vein.tfg.surface_tetrahedrite": "Tetrahedrite (Surface)",
-"ore_vein.tfg.mars_hematite": "(Placeholder Vein)",
+"ore_vein.tfg.deep_chromite": "Deep Redstone and Chromite",
+"ore_vein.tfg.deep_pitchblende": "Deep Pitchblende",
+"ore_vein.tfg.deep_sheldonite": "Deep Sheldonite",
+"ore_vein.tfg.mars_almandine": "Alamdine",
+"ore_vein.tfg.mars_apatite": "Apatite",
+"ore_vein.tfg.mars_coal": "Coal and Iron",
+"ore_vein.tfg.mars_copper": "Copper",
+"ore_vein.tfg.mars_galena": "Galena",
+"ore_vein.tfg.mars_gold": "Iron and Gold",
+"ore_vein.tfg.mars_lubricant": "Lubricant",
+"ore_vein.tfg.mars_neodynium": "Neodymium",
+"ore_vein.tfg.mars_nickel": "Nickel and Cobaltite",
+"ore_vein.tfg.mars_pitchblende": "Pitchblende and Thorium",
+"ore_vein.tfg.mars_quartzite": "Quartzite and Asbestos",
+"ore_vein.tfg.mars_salt": "Salt and Lepidolite",
+"ore_vein.tfg.mars_sapphire": "Sapphire",
+"ore_vein.tfg.mars_stibnite": "Copper and Stibnite",
+"ore_vein.tfg.mars_sulfur": "Sulfur and Pyritie",
+"ore_vein.tfg.mars_tantalite": "Grossular, Spessartine and Tantalite",
```

Comment:
This and the nickel one need to be shorter, otherwise it won't fit on the page in the field guide